frontend/packages/core/src/Charts/tests/helpers.test.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+  calculateDomainEdges,
+  calculateTicks,
+  getLeftSideAndIntervalForTicks,
+  getMinAndMaxOfRangeUsingKey,
+  isoTimeFormatter,
+} from "../helpers";
+
+describe("helper functions for charts", () => {
+  describe("isoTimeFormatter", () => {
+    test("a timestamp", () => {
+      expect(isoTimeFormatter(60)).toBe("1970-01-01T00:00:00.060Z");
+    });
+  });
+
+  const testDataMeowWoof = [
+    {
+      meow: "1",
+      woof: "2",
+    },
+    {
+      meow: "100",
+      woof: "2",
+    },
+    {
+      meow: "50",
+      woof: "2",
+    },
+  ];
+  describe("getMinAndMaxOfRangeUsingKey", () => {
+    test("the key exists in the data", () => {
```

Comment:
nit: when writing these test blocks it helps to think about how this will read
```
> helper functions for charts
>> getMinAndMaxOfRangeUsingKey
>>> the key exists in the data ✅
```

Generally you'd want some information about expectations in the test names so it's easier to debug from a higher level. Reading through these tests it seems like what you're trying to assert is that different range lookup cases are handled without errors. Is that right?